.env.example

```diff
@@ -5,16 +5,12 @@
 ## 1 = Mainnet
 CHAIN_ID=31337
 
-## Keys used to encrypt secrets and sign ETH transactions
+## Key used to encrypt secrets and sign ETH transactions
```

Comment:
The intent is for end users running nodes to be using the quickstart repo and not this one right? I'm thinking we should really gear the .env in that quickstart repo toward real users (so no hardhat defaults or comments). Commenting here because there's no pr for that other repo.